coderd/oauth2.go

```diff
@@ -566,7 +566,13 @@ func (api *API) postOAuth2ClientRegistration(rw http.ResponseWriter, r *http.Req
 		RegistrationClientUri:   sql.NullString{String: fmt.Sprintf("%s/oauth2/clients/%s", api.AccessURL.String(), clientID), Valid: true},
 	})
 	if err != nil {
-		api.Logger.Error(ctx, "failed to store oauth2 client registration", slog.Error(err))
+		clientName := generateClientName(req)
+		api.Logger.Error(ctx, "failed to store oauth2 client registration",
+			slog.Error(err),
+			slog.F("client_name", clientName),
+			slog.F("client_id", clientID.String()),
+			slog.F("redirect_uris", req.RedirectURIs))
+
```

Comment:
nit: wrong stack

Reply:
Do you mean that the error should go last? I don't think I'm understanding the nit here 😅

Reply:
It just seemed like this particular hunk is unrelated to this PR, but please correct me if I'm wrong!